docs/UI/SetGlobalProperties/index.md

```diff
@@ -9,30 +9,98 @@ Sender
 Purpose
 : Set the UI properties of an application.
 
+### Description
+SDL requests to set-up the data for VR help layout, the name and icon for in-application menu and the properties of the touchscreen keyboard.
+
+The request may arrive for the application whatever being active or in background on HMI (depends on Policy Table permissions applicable to mobile application request, by default allowed to operate in all HMI levels except of NONE).
+
+SDL sends _SetGlobalProperties_request_ with specific _vrHelp_ and _vrHelpTitle_ values to HMI in next cases:   
+1)	In case mobile app sends the very first _SetGlobalProperties_request_ in current ignition cycle
+with `<VRHelp>` and `<VRHelpTitle>` params, SDL transfers `<VRHelp>` and `<VRHelpTitle>`.   
+2)	In case mobile app sends the very first _SetGlobalProperties_request_ in current ignition cycle
+without `<VRHelp>` and `<VRHelpTitle>` params, SDL generates default values of `<VRHelp>` and `<VRHelpTitle>` and transfers them.   
+3)	In case mobile app sends the next (not first) _SetGlobalProperties_request_ within the same ignition cycle with `<VRHelp>` and `<VRHelpTitle>` params, SDL transfers them.   
+4)	In case mobile app the next (not first) _SetGlobalProperties_request_ within the same ignition cycle without `<VRHelp>` and `<VRHelpTitle>` params, SDL omits (not sends) `<VRHelp>` and `<VRHelpTitle>` to HMI.   
+5)	In case SDL resumes `<VRHelp>` and `<VRHelpTitle>` during Data resumption, SDL must not send default values of VR at the nexts (not first) _SetGlobalProperties_ requests.   
+6) In case mobile app sends _SetGlobalProperties_request_ to SDL:   
+- with both valid values of _autoCompleteList_ and _autoCompleteText_ params, _SDL must_:   
+- [x] tranfer _SetGlobalProperties_request_ with _autoCompleteList_ param and without (omitted) _autoCompleteText_ param to HMI;   
+- [x] respond with \<resultCode_received_from_HMI> to mobile app.   
```

Comment:
@jacobkeeler, right. I see. Could you please tell if there something like list of supported by portal markdown features. Because as I understand Guthub Task lists https://guides.github.com/features/mastering-markdown/ is not actual  for portal.